app/services/clients/base-client/wrapper.js

```diff
@@ -49,21 +47,13 @@ module.exports = function (method, verb) {
         body = transform ? transform(body) : body
         resolve(body)
       } else {
-        if (handleError === 'new') {
-          let errors = lodash.get(body, 'message') || lodash.get(body, 'errors')
-          if (errors && errors.constructor.name === 'Array') errors = errors.join(', ')
-          const err = new Error(errors || body || 'Unknown error')
-          err.errorCode = response.statusCode
-          err.errorIdentifier = lodash.get(body, 'error_identifier')
-          err.reason = lodash.get(body, 'reason')
-          reject(err)
-        } else {
-          // eslint-disable-next-line
-          reject({
-            errorCode: response.statusCode,
-            message: response.body
-          })
-        }
+        let errors = lodash.get(body, 'message') || lodash.get(body, 'errors')
+        if (errors && errors.constructor.name === 'Array') errors = errors.join(', ')
```

Comment:
This is old code that sat in a condition you've removed so entirely optional but this feels a bit janky
```suggestion
        if (errors && Array.isArray(errors)) errors = errors.join(', ')
```

Reply:
Thanks, will use your suggestion in an upcoming PR